config/locales/alchemy.en.yml

```diff
@@ -289,6 +289,7 @@ en:
     content_definition_missing: "Warning: Content is missing its definition. Please check the elements.yml"
     element_definition_missing: "WARNING! Missing element definition. Please check your elements.yml file."
     page_definition_missing: "WARNING! Missing page layout definition. Please check your page_layouts.yml file."
+    page_roles_denial: "WARNING! Your user role does not allow you to edit this page. Please check your page_layouts.yml file."
```

Comment:
Why introduce another translation. We could use the same as in the controller message